.github/workflows/build-publish-develop.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Build Chainlink and push image with latest develop tag'
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build-publish-chainlink-develop:
+    runs-on: ubuntu-20.04
+    environment: build-develop
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+
+      - name: Build, sign and publish chainlink image
+        uses: ./.github/actions/build-sign-publish-chainlink
+        with:
+          publish: true
+          image-name: chainlink
```

Comment:
```suggestion
```

This should no longer be needed

Reply:
Good catch, removed other instance as well as it's unused in the code path and the test build can use the default value (not publishing)